src/main/java/com/scriptbasic/syntax/BasicSyntaxAnalyzer.java

```diff
@@ -50,12 +55,24 @@ public BuildableProgram analyze() throws AnalysisException {
                     buildableProgram.addCommand(newCommand);
                 }
             }
+            if(additionalCommands!=null) {
+                additionalCommands.forEach(command -> buildableProgram.addCommand(command));
```

Comment:
it is shorter to use method reference instead of a lambda expression

Reply:
Fixed